Christian branch

ChristianTW opened the pull request:
The only things that should be changed is my scene, the torch, brazier, lantern, stone door, and arrow prefabs, and the StarterArea scene. I added a water material and set the water to that material.

I also changed the SpawnArrowWASD script, SpawnArrowVR script, SecondTargetChecker script, ExplosiveBarrel script, AudioManager, and I made the arrow impact script.

I have ambiance for the starting area, music for the puzzles, fire sounds, arrow impact sounds, footsteps, and a win sound for solving a puzzle. I changed the sound when an arrow hits a target to be more distinct. I added an explosion sound that will play when the barrel explodes.

I fixed the arrow sounds so each arrow only plays one sound on their first impact.

The bow firing sound should also play in VR now, but I wasn't able to test it.

There are still some errors with the stone door in the ThroughTheFireAndTheFlames scene, however the scene itself should be unchanged.

ldroberts02 committed:
Pre Merge to Luke Branch

ldroberts02 committed:
Merge remote-tracking branch 'origin/aiden-branch' into luke-branch

ldroberts02 committed:
Merge remote-tracking branch 'origin/colby-branch' into luke-branch

ldroberts02 committed:
Finished with menus and arrows, pre merge.

ldroberts02 committed:
Merge remote-tracking branch 'origin/main' into luke-branch

ldroberts02 committed:
Pause menu work, new arrow wall stick

Modified hands and post processing for pause menu, changed how arrows are calculated in walls.

ldroberts02 committed:
Pushed Arrow Change

Commit:
Worked on torch audio

ldroberts02 committed:
Pre Merge

ldroberts02 committed:
Merge branch 'main' into luke-branch

ldroberts02 committed:
Further Work before merge

ldroberts02 committed:
Merge branch 'colby-branch' into luke-branch

Commit:
Fire sound added to torch, wall torch, lantern, and braizer prefab

ldroberts02 committed:
Merge remote-tracking branch 'origin/colby-branch' into luke-branch

Commit:
Merge remote-tracking branch 'origin/main' into christian-branch

ldroberts02 committed:
Auto stash before merge of "luke-branch" and "origin/luke-branch"

ldroberts02 committed:
Fix Arrow Issue

Commit:
Arrow impact sound works now

Commit:
Merge branch 'christian-branch' of https://github.com/EricWRogers/vr_workshop_2024 into christian-branch

Commit:
Merge branch 'colby-branch' into christian-branch

ldroberts02 committed:
Cleaned up build scene, lit

ldroberts02 committed:
Moved changes to separate scene, fixed arrow but had to settle for issue

The arrow has an issue, when on stick it disables the box collider to ensure you cant stick arrows to arrows or player colliding with them, but it prevents the targets from activating. This function was disabled until i can get with the target programmer.

ldroberts02 committed:
Fix The Arrows, menus

ldroberts02 committed:
Merge branch 'main' into luke-branch

ldroberts02 committed:
Merge branch 'main' into luke-branch

ldroberts02 committed:
Scene updated, arrows stick, final

Commit:
Added cave music and stone door sound, added fire sound to fire arrow

Added it to arrows lit on fire specifically

ldroberts02 committed:
More work on scenes

Commit:
Script should make a different sound when impacting a target

Commit:
Added target to whitebox for testing

The whitebox is in my folder, so it won't cause issues

Commit:
Merge remote-tracking branch 'origin/luke-branch' into christian-branch

Commit:
Luke stuff merged into mine, door sound added, target sound is 2D

Commit:
Added win sound, added checkor for when door animation ends to script

Commit:
Added sound for end of door opening

Commit:
Tweaked audio levels

Fire sound Is still a bit too loud

Commit:
Door sound uses audio source

Not Audio Manager

ldroberts02 committed:
Stripping previous Luke Work from Christian-Branch

ldroberts02 committed:
Merge Main into Christian-Branch

ldroberts02 committed:
Put main's scenes into christian-branch

ldroberts02 committed:
Merge branch 'main' into christian-branch

ldroberts02 committed:
Put Arrow impact script back into arrow prefab, like christian had prior

Commit:
Merge branch 'christian-branch' of https://github.com/EricWRogers/vr_workshop_2024 into christian-branch

Aiden-Thacker reviewed (changes requested):
Currently you have merge conflicts that we need to resolve before we can merge your branch to main. If you know how to resolve the merge conflict go ahead and do it and dm the server chat and I will merge it to main. If you do not know how to do it I can help in class tomorrow

Colby626 committed:
Merge in favor of Main

Colby626 commented:
So I fixed the merge conflict, it can pull into main fine now. Whenever you open your branch it will tell you you need to fix lighting data. Select no, it is fixed in main and we don't want another conflict. However, your audio I think for the arrows plays too often (might be because it is impacting a bunch of times instead of sticking) and there is no shot I want that in build. It gets super loud and you can hear it from anywhere. Generally I think the fire is too loud, a soft crackle makes more sense.

Commit:
Added new target hit sound, added stone door crash, added water splash

Water splash not implemented properly yet. Win jingle now attached to door spawn/ puzzle win.

Commit:
Merge branch 'christian-branch' of https://github.com/EricWRogers/vr_workshop_2024 into christian-branch

Commit:
Arrow splashes in water, explosive barrels make sound, fixed arrow sound

Commit:
Firing arrows should play sound in VR

LiamYork9 commented:
Closed so you can make a new one
